src/RadialMenu/RadialMenu/ALRadialMenu.cs

```diff
@@ -100,10 +111,12 @@ private void CommonInit()
                 CancelsTouchesInView = _overlayCancelsTouchesInView
             };
 
-            _overlayView = new PassThroughView(UIScreen.MainScreen.Bounds) {
+            _overlayView = new PassThroughView(UIScreen.MainScreen.Bounds)
+            {
                 PassThroughTouchEvents = !_overlayCancelsTouchesInView
             };
             _overlayView.AddGestureRecognizer(_dismissGesture);
+            _overlayView.OnOverlayPointInside += OverlayView_OnOverlayPointInside;
```

Comment:
Why is this event needed?

Reply:
Setting SetOverlayCancelsTouchesInView to false would cause the tap/dismiss gesture to never be recognized thus never closing the menu when tapping the overlay.
I chose to fix this by catching the new event from the subview and handle the rest in the ALRadialMenu class. Do you see a more straight forward fix to this issue?